pidev/src/main/java/models/Device.java

```diff
@@ -1,10 +1,10 @@
 package models;
 
 public class Device {
-    private String deviceName;
```

Comment:
Might want to keep these private? 
Is there a way to merge everything but this?